src/cls/rgw/cls_rgw_types.cc

```diff
@@ -4,12 +4,14 @@
 #include "cls/rgw/cls_rgw_types.h"
 #include "common/ceph_json.h"
 #include "include/utime.h"
+#include<list>
 
 using std::list;
 using std::string;
 
 using ceph::bufferlist;
 using ceph::Formatter;
+using namespace std;
```

Comment:
can we please use std:: where required, rather than opening namespace std in the entire file?